core/src/main/scala/org/apache/spark/resource/ResourceProfileManager.scala

```diff
@@ -54,6 +54,7 @@ private[spark] class ResourceProfileManager(sparkConf: SparkConf,
   private val master = sparkConf.getOption("spark.master")
   private val isYarn = master.isDefined && master.get.equals("yarn")
   private val isK8s = master.isDefined && master.get.startsWith("k8s://")
+  private val isStandalone = master.isDefined && master.get.startsWith("spark://")
```

Comment:
what about local cluster here?  it would be nice to update description if adding support for it as well. Otherwise remove code from ResourceProfile